### ISSUE TITLE
Add a 'Sign in with MixCloud' button on homepage

### ISSUE THREAD

**Issue by roperzh** [roperzh/convo#1]
If the user is logged out, the app should display a button allowing to sign up with mixcloud

**Comment by roperzh** [comment]
Fixed with 127840ba5a0bb581af0f78d4fff0e64781108865

Styles are still missing, but it is a better idea to tackle them in another issue once the whole app design is finished.
